src/vendor/requests.js

```diff
@@ -0,0 +1,103 @@
+/* global fetch */
+import { parse } from 'query-string';
+import { leaves, toPairs } from './vectors';
+import { Log } from './logs';
+import { Data } from './Data';
+import { exists, isArray, isString, toArray } from './utils';
+import strings from './strings';
+
+function fromQueryString(query) {
+  const decode = v => {
+    if (isArray(v)) return v.map(decode);
+
+    if (v === null || v === '') return true;
```

Comment:
Not adding braces for if clauses can be somewhat of a footgun if you go back later and add a second statement to the if clause without remembering to add the curly braces, the code will break in unexpected ways. Also, depending on the line length, the linter might put the statement in a newline. I typically always add braces for maintainability.

```js
if (something) {
  doSomething();
}
```